ddr/Makefile

```diff
@@ -26,7 +26,7 @@ include $(top_srcdir)/omrmakefiles/configure.mk
 
 OMR_TOOLCHAIN ?= gcc
 
-ifneq (,$(filter gcc xlc,$(OMR_TOOLCHAIN)))
+ifneq (,$(filter gcc xlc openxl,$(OMR_TOOLCHAIN)))
```

Comment:
Do we care about non-cmake builds with OpenXL? If not, the remaining changes should be removed.